test/test_material.py

```diff
@@ -13,11 +13,15 @@ def setUp(self):
         # Append current path to locate files
         self.material = pywavefront.material.Material(prepend_dir('material'))
         self.material.set_texture(prepend_dir('4x4.png'))
+        self.material.set_texture_ambient(prepend_dir('4x4.png'))
 
     def testSetTexture(self):
         """Running set_texture should set a texture."""
-        self.assertEqual(self.material.texture.__class__,
-                pywavefront.texture.Texture)
+        self.assertEqual(self.material.texture.__class__, pywavefront.texture.Texture)
```

Comment:
Is this change only indentation?

Probably better to have this and the following line broken up into separate lines

Reply:
Looks like it was fixed because it had "Wrong continued indentation". I can split them up again. Probably happened because I pasted it back in at some point and my editor auto formats on pasting.